opta/flyte.yaml

```diff
@@ -43,7 +43,8 @@ modules:
     links:
       - s3: [ "write" ]
   - type: helm-chart
-    chart: "../helm" # NOTE: relative path to chart
+    repository: https://flyteorg.github.io/flyte
+    chart: flyte-core
```

Comment:
flyte
```suggestion
    chart: flyte
```

Reply:
Not according to your docs: https://artifacthub.io/packages/helm/flyte/flyte

Reply:
Thank you... will fix the docs..